pkgs/tools/text/Gazou/default.nix

```diff
@@ -0,0 +1,29 @@
+# gazou.nix
+{ lib, stdenv, fetchFromGitHub, cmake, leptonica, pkgconfig, qtbase, qtx11extras, tesseract, wrapQtAppsHook }:
+
+stdenv.mkDerivation {
+  pname = "gazou";
+  version = "unstable-2023-05-14";
```

Comment:
As there are releases available, please add a comment explaining why you use a snapshot.

Reply:
Current release does not build properly or have proper support for X11. Verified w/ some community members this was the case as well.

Reply:
I have seen the discussion in the chat, but not the poeple reading this drv in a week or month, please provide a comment, or add patches that would fix the issue for the release (with a comment).

Do the comment in the file, not here.

Reply:
I can simply provide the comment note to the right of the used hash to indicate the reason?

Reply:
Personally I'd provide it close to the `version` rather than the `src`. As the `src` just follows the `version` usually.